server/views/components/facility-promo/facility-promo.njk

```diff
@@ -0,0 +1,33 @@
+{% if data.sizes %}
```

Comment:
I appreciate I probably wrote this originally, but it could probably be

`{% set sizes = data.sizes if data.sizes else '100vw' %}`

I'm all about the ternaries today.